collections/_article/dev-snapshot-godot-4-6-dev-4.md

```diff
@@ -0,0 +1,140 @@
+
+<video autoplay loop muted playsinline title="Project manager navigation">
+  <source src="/storage/blog/dev-snapshot-godot-4-6-dev-4/project-manager-nagivation.mp4?1" type="video/mp4">
+</video>
+
+Next is [Malcolm Anderson](https://github.com/Meorge), who implemented functionality for opening the project in the file explorer as a "Show in File Manager" button ([GH-111624](https://github.com/godotengine/godot/pull/111624)). Unfortunately, while a highly-requested feature, it proved to be too cluttered and was slated for a revert. Not being one to throw out the baby with the bathwater, [Tomasz Chabora](https://github.com/KoBeWi) saved the functionality by reimplementing it in the form of a newly-added right-click menu:
+
+<video autoplay loop muted playsinline title="Project manager navigation">
+  <source src="/storage/blog/dev-snapshot-godot-4-6-dev-4/project-manager-right-click.mp4?1" type="video/mp4">
+</video>
+
+Speaking of Tomasz, he's here to round things off with yet another highly-requested feature: the ability to modify editor settings within the project manager ([GH-82212](https://github.com/godotengine/godot/pull/82212)). Now opening a project just to make general adjustments is no longer necessary, as it can all be handled in the much more lightweight context of the project manager:
+
+<video autoplay loop muted playsinline title="Project manager navigation">
+  <source src="/storage/blog/dev-snapshot-godot-4-6-dev-4/project-manager-editor-settings.mp4?1" type="video/mp4">
+</video>
+
+### Buildsystem: Support built-in profilers
+
+<div class="card card-warning">
+<p>This is unapologetically super-nerd territory, and strictly targeting those who already know what this is talking about. Everyone else, feel free to jump to the curated highlights instead.</p>
+</div>
+
+It's rare for our blog posts to mention the buildsystem in any capacity — let alone as a featured highlight — but this is a very special exception: Godot can now natively support built-in profilers [GH-104851](https://github.com/godotengine/godot/pull/104851)! Note that this is separate from [the engine's profiler](https://docs.godotengine.org/en/latest/tutorials/scripting/debug/the_profiler.html), as that's suited for projects running *in* the engine, rather than the engine *itself*.
+
+Thanks to the efforts of [Lukas Tenbrink](https://github.com/Ivorforce), building off the incredible foundation of [Andrea Catania](https://github.com/AndreaCatania)'s [GodotTracy module](https://github.com/AndreaCatania/godot_tracy), engine developers will no longer need to manually integrate (and constantly re-integrate) profiling logic to the engine. Instead: they merely need to pass the appropriate path to `profiler_path`, and our buildsystem will automatically detect and integrate the given profiler (currently supports [tracy](https://github.com/wolfpld/tracy) and [perfetto](https://perfetto.dev/)).
```

Comment:
This is not really true. I've looked at their implementation at first, but ultimately built something entirely different from scratch, since what they made wasn't really what we needed in the engine itself.